comments/models.js

```diff
@@ -25,11 +25,23 @@ exports.insertComment = (username, body, article_id) => {
     RETURNING *
     ;`
 
-    return checkArticleExists(article_id).then(() => {
+    return checkExists('articles', 'article_id', article_id).then(() => {
         return db.query(queryStr, [body, article_id, username])
         .then(({ rows }) => {
             return rows[0]
         })
     })
     
-}
\ No newline at end of file
+}
+
+exports.removeCommentById = (comment_id) => {
+    const queryStr = `
+        DELETE FROM comments
+        WHERE comment_id = $1
+    ;`
```

Comment:
If you used a `RETURNING *` here you could tell if it existed based on the number of rows returned